vspace/pom.xml

```diff
@@ -183,7 +183,14 @@
             <version>4.12</version>
             <scope>test</scope>
         </dependency>
-
+        <!-- https://mvnrepository.com/artifact/org.springframework/spring-test -->
+        <dependency>
+            <groupId>org.springframework</groupId>
+            <artifactId>spring-test</artifactId>
+            <scope>test</scope>
+        </dependency>
```

Comment:
And do you really need this?

Reply:
Couldnt find a way to pass custom value for page size without using this library.